### PR TITLE
Create editExerciseComment mutation and update some snapshots for tests

### PR DESCRIPTION
**Pull request by HS-90**
**Description:** relates to https://github.com/garageScript/c0d3-app/issues/2400

This PR adds a graphQL mutation that will handle editing of exercise comments as part of the discussions feature in C0D3 Dojo.
To test:
Use an existing exerciseComment `id` or create a new one in graphQL like so:

1. Go to 'api/graphql'
2. write a query as follows:
```
mutation AddExerciseComment{   
       addExerciseComment(exerciseId:1, content:"Cool")
             { id 
              exerciseId 
              content   } 
}
```

3. You should receive a response such as: 
```
{   "data": {
        "addExerciseComment": { 
            "id": 3, 
            "exerciseId": 1, 
            "content": "Cool" }   
                  }
 }
```
image

4. Remember/save the `id` that is generated/assigned to the newly created `exerciseComment`

5. Use this id to run a query such as:
```
mutation EditExerciseComment{   
       editExerciseComment( id: 3, content: "I edited this comment")
             { id 
              createdAt
              exerciseId
              parentId
              content   } 
}
```

If successful, there should be a result similar to the following:
editted exercise comment

### CODE REVIEW

**Comment by flacial** [comment]
It shouldn't update any snapshots. Did you merge upstream changes and update the packages?

File: graphql/resolvers/exerciseCommentCrud.ts [hunk 1]
```diff
@@ -44,3 +45,28 @@ export const addExerciseComment = async (
...
+  const exerciseComment = await prisma.exerciseComment.findUnique({
+    where: {
+      id
+    }
+  })
+
+  if (exerciseComment?.authorId !== authorId)
+    throw new Error('Comment is not by user')
+
+  return prisma.exerciseComment.update({
```
Comment (flacial) [inline graphql/resolvers/exerciseCommentCrud.ts:66]: It would be nice to include the date too.

File: graphql/resolvers/exerciseCommentCrud.ts [hunk 2]
```diff
@@ -44,3 +45,28 @@ export const addExerciseComment = async (
     data: { authorId, content, exerciseId, parentId, userPic }
   })
 }
+
+export const editExerciseComment = async (
```
Comment (flacial) [inline graphql/resolvers/exerciseCommentCrud.ts:49]: Can't we wrap this with [WithUserContainer](https://github.com/garageScript/c0d3-app/blob/7ab5efcd9c2b726846cc747ca83e846712ab27cf/containers/withUserContainer.ts) higher order function?

**Comment by HS-90** [comment]
> [It shouldn't update any snapshots](https://github.com/garageScript/c0d3-app/actions/runs/4049052604/jobs/6965727153) because they were updated and merged in this [PR](https://github.com/garageScript/c0d3-app/pull/2725). Did you merge upstream changes and update the packages?

I did update my fork and did 'git pull' in my local repo before submitting this PR. I had to update the snapshots in my local with 'yarn test -u' otherwise the snapshots failed. I'll have to look further into why it's failing the test.

File: graphql/resolvers/exerciseCommentCrud.ts (hunk 1 above)
Comment (HS-90) [reply graphql/resolvers/exerciseCommentCrud.ts:66]: Actually, I though prisma updates that automatically in the  "updatedAt" field. If that's the case we wont need to manually update the time right?

Comment (flacial) [reply graphql/resolvers/exerciseCommentCrud.ts]: You're right.

**Review by flacial — APPROVED** [review]

**Review by anthonykhoa — APPROVED** [review]

**Comment by anthonykhoa** [comment]
Thanks for the detailed test instructions. LGTM! 👍

**Review by SlyBouhafs — APPROVED** [review]

**Pull request merged**